quic/s2n-quic-qns/src/perf.rs

```diff
@@ -278,8 +305,12 @@ impl Counters {
         let duration = self.timeout.swap(0, Ordering::Relaxed);
         let duration = Duration::from_nanos(duration);
         let pto_count = self.pto_count.swap(0, Ordering::Relaxed);
+        let max_pacing_rate = self.max_pacing_rate.swap(0, Ordering::Relaxed);
+        let max_pacing_rate = rate(max_pacing_rate, Duration::from_secs(1));
```

Comment:
```suggestion
        let max_pacing_rate = rate(max_pacing_rate, duration);
```

Reply:
It actually needs to be `Duration::from_secs(1)` because the pacing rate and delivery rate events output the value as "bytes/sec", so regardless of the reporting duration it is always bytes per 1 second